doc/events.md

```diff
@@ -0,0 +1,126 @@
+# Events
+
+## Introduction
+Events are messages about actions that have occurred over LXD. Using the API endpoint `1.0/events` directly or via
+`lxd monitor` will connect to a WebSocket through which logs and lifecycle messages will be streamed.
+
```

Comment:
We should add an extra section here for `Event types` which covers the 3 types of events:
 - Logging (shows all logging messages regardless of the server logging level)
 - Operation (shows all ongoing operations from creation to completion including updates to their state and progress metadata)
 - Lifecycle